pandas/_libs/window/aggregations.pyx

```diff
@@ -136,7 +136,7 @@ cdef inline void remove_sum(float64_t val, int64_t *nobs, float64_t *sum_x,
         sum_x[0] = t
 
 
-def roll_sum(ndarray[float64_t] values, ndarray[int64_t] start,
+def roll_sum(const float64_t[:] values, ndarray[int64_t] start,
```

Comment:
Can start be a memoryview?

Reply:
No (not yet). start/end go through functions that still expect a ndarray.